packages/ui/ui/rating.tsx

```
@@ -0,0 +1,81 @@
+	rating,
+	maxRating = 5,
+	size,
+	className,
+	ref,
+	...props
+}: RatingProps) {
+	return (
+		<div
+			className={cn(ratingVariants({ size }), className)}
+			role="img"
+			aria-label={`Rating: ${rating} out of ${maxRating}`}
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)